WC-Local-Pickup/woocommerce-local-pickup-time
# Adds New Ready for Pickup Order Status & Customer Email

### All Submissions:






### Changes proposed in this Pull Request:



Closes #7 .

- Adds a new Order Status of "Ready for Pickup".
- Adds a new email actions & email templates used when an order is marked "Ready for Pickup".
- Updates code quality checks & local development tooling.

### How to test the changes in this Pull Request:

1. Login to the WordPress Dashboard.
2. Edit a WooCommerce Order or choose to bulk edit orders.
3. A new "Ready for Pickup" Order Status should be available.
4. When changing the Order Status to "Ready for Pickup" an email should be sent to the customer.
5. In the WooCommerce Email settings there is a new "Ready for Pickup" email that can be changes and/or disabled.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.

- Adds new "Ready for Pickup" order status and email notification.
